mule-packager/src/test/java/org/mule/tools/api/PackageBuilderTest.java

```diff
@@ -58,11 +57,8 @@
 
   @Before
   public void setUp() throws IOException {
-    this.packageBuilder = new PackageBuilder();
-    this.packageBuilderSpy = spy(PackageBuilder.class);
+    this.packageBuilder = new MulePackageBuilder();
```

Comment:
It seems you're testing MulePackageBuilder please rename MulePackageBuilderTest
Also please generate a DomainBundlePackageBuilderTest